### PR TITLE
fixed Eigen::Constant type reference issue

### PR DESCRIPTION
**Pull request by ct-clmsn**
fixes type resolution issue when calling the Eigen static Constant method.

### CODE REVIEW

**Comment by ct-clmsn** [comment]
found a small hpx header file missing issue. fixed the issue in hpx and added a [PR](https://github.com/STEllAR-GROUP/hpx/pull/2892).

File: src/primitives/add_operation.cpp [hunk 1]
```diff
@@ -175,7 +175,7 @@ namespace phylanx { namespace primitives
     // implement '+' for all possible combinations of lhs and rhs
     hpx::future<ir::node_data<double>> add_operation::eval()
     {
-        return hpx::dataflow(hpx::util::unwrapping(
+        return hpx::dataflow(hpx::util::unwrapped(
```
Comment (hkaiser) [inline src/primitives/add_operation.cpp]: This has been recently renamed. What version of HPX should we target with Phylanx? Would it be ok to stick with HPX master for the foreseeable future?

Comment (ct-clmsn) [reply src/primitives/add_operation.cpp]: my poor git utilization strikes again! apologies, will amend.

Comment (ct-clmsn) [reply src/primitives/add_operation.cpp]: @hkaiser do i need to do anything to drop that last commit for 'hpx::util::unwrapping'?

Comment (hkaiser) [reply src/primitives/add_operation.cpp]: @ct-clmsn it should be fine to remove the commit from your local branch (reset to b18e6332b070e17a7f32297ea0a5f88e91d765f8) and force-push to github.

**Comment by ct-clmsn** [comment]
@hkaiser should be rolled back now. thanks!

**Pull request merged**